modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/domain/delegatedStake.scala

```diff
@@ -0,0 +1,12 @@
+package io.constellationnetwork.dag.l0.domain
+
+import io.constellationnetwork.schema.delegatedStake.UpdateDelegatedStake
+import io.constellationnetwork.security.signature.Signed
+
+object delegatedStake {
+
+  sealed trait DelegatedStakeOutput
```

Comment:
This is maybe not the best naming here, although I do appreciate what you're trying to do by wrapping these Signed[T] types in an event class.

Since these are essentially wrappers around the submit endpoints, maybe Request or Event is a better name? I think output is a little confusing here.